docs-site/content/docs/the-app/models.md

```diff
@@ -165,6 +165,19 @@ Loco is a migration-first framework, similar to Rails. Which means that when you
 
 This enforces _everything-as-code_, _reproducibility_ and _atomicity_, where no knowledge of the schema goes missing. 
 
+### Down Migrations
+
+If you realize that you made a mistake, you can always undo the migration. This will undo the changes made by the migration (assuming that you added the appropriate code for `down` in the migration).
+
+```
+$ cargo loco db down
+```
```

Comment:
Please move this (also the second command) command to take it from [snipdoc. yml](https://github.com/loco-rs/loco/blob/master/snipdoc.yml).
This is a tool that we built to improve our documentation. See how to use this tool [here](https://github.com/kaplanelad/snipdoc)

Reply:
I took a crack at it but it also went after some other code snippets. What should I do there?